README.md

```diff
@@ -160,6 +160,12 @@ Option to define a custom file association on Windows.
 Caution: when you use `win.nsiTemplate` option, `win.fileAssociation` option should only work
 if the custom nsi template is based on the original one.
 
+### `win.encoding` *( optional )*
+Option to define a custom encoding for output template.nsi on Windows.
+Default value is 'UTF-8'.
+For available values see [ashtuchkin/iconv-lite's page](https://github.com/ashtuchkin/iconv-lite/wiki/Supported-Encodings).
+The packager.json config should always be UTF8 encoded.
```

Comment:
Does it have to contain the utf8 BOM (EF BB BF), or will it also work without it?
What ever is supported should probably be stated explicitly in the doc.